src/test/java/io/vertx/core/spi/tracing/LocalEventBusTracerTest.java

```diff
@@ -32,15 +32,15 @@ public void testInboundInterceptor() throws Exception {
     tracer = new VertxTracer() {};
     vertx2.eventBus().addInboundInterceptor(deliveryCtx -> {
       ContextInternal ctx = (ContextInternal) vertx.getOrCreateContext();
-      ctx.localContextData().put("key", "val");
+//      ctx.putLocal(FakeTracer.ACTIVE_SCOPE_KEY, "val");
```

Comment:
What's wrong here?

Reply:
test is actually ignored :-) so previous code will not work anymore when the deprecated code will be removed, I sketched what should be correct more or less